fdb-record-layer-jmh/fdb-record-layer-jmh.gradle

```diff
@@ -37,13 +37,28 @@ sourceSets {
 def coreProject = ":${ext.coreProjectName}"
 dependencies {
     api project(coreProject)
+    implementation project(":fdb-record-layer-lucene")
```

Comment:
(I don't know gradle....) no need of a wildcard '*' for groupids before the ':' ?